src/libsync/theme.cpp

```diff
@@ -414,10 +414,19 @@ QString Theme::aboutVersions(Theme::VersionFormat format) const
             gitUrl = gitSHA1(format) + br;
         }
     }
+
+
+#if defined(OC_PLUGIN_DIR)
+    const QString pluginDirPath = QDir(QCoreApplication::applicationDirPath()).filePath(QStringLiteral(OC_PLUGIN_DIR));
+#else
+    const QString pluginDirPath = QStringLiteral("not set");
```

Comment:
Can we just not mention it at all?
OC_PLUGIN_DIR is just a workaround for the weird package layout we provide on linux.

Reply:
We intentionally introduced this in 4fbcf5cd. We should make the path available _somewhere_ at least.

Reply:
You clarified you just want to remove the else branch. Sure, let's do that.